scripts/prepare_heat_demand.py

```diff
@@ -0,0 +1,305 @@
+# coding: utf-8
+r"""
+Inputs
+-------
+in_path1 : str
+    path of input directory with raw data
```

Comment:
Following #88, we would like to include the filenames that are defined in Snakefile here, to make it easier to learn from the documentation about what files go in and out.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/d26d822fd1b59e4ba68c97ed10e1173a13242434.